### PR TITLE
Update routes.js to send a response for every request

### PR DESCRIPTION
**Pull request by bhavyaagg**
Signed-off-by: Bhavya Aggarwal <bhavyaaggarwal1209@gmail.com>

I have updated routes.js to send a response for every route.

### CODE REVIEW

**Comment by bhavyaagg** [comment]
Can you please review this @agentmilindu ?

**Comment by bhavyaagg** [comment]
@psnmissaka Thanks for reviewing my PR. Sorry but i am unwell today. Will update my PRs tomorrow morning for sure.

**Comment by agentmilindu** [comment]
@psnmissaka @tharindupr Can we merge this PR?

CC @padamchopra @iammosespaulr

**Comment by psnmissaka** [comment]
@agentmilindu once the issues are fixed and PR is updated, i can test the API end points and if everything works we can merge :)

**Comment by padamchopra** [comment]
@psnmissaka , an appropriate answer for that query can be found in this [stack overflow query](https://stackoverflow.com/questions/242813/when-to-use-double-or-single-quotes-in-javascript?page=1&tab=oldest#tab-top). Personally, I feel that back-tick is the best way as it allows using double quotes and single quotes within the string. But then the codacy tests won't pass 😟

File: stackle_api/app/routes.js [hunk 1]
```diff
@@ -12,198 +12,213 @@ var Reply = postModels.Reply;
...
-	})
-
-	app.get('/api/notifications', function (req, res) { });
-
-	app.get('/*', function (req, res) {
-		res.sendfile('./public/404.html');
-	});
+    //api
+    app.get('/api/login/', function (req, res) {
+        res.status(501).send("Not Implemented")
```
Comment (iammosespaulr) [inline stackle_api/app/routes.js]: add semi-colon to fix one codacy issue

File: stackle_api/app/routes.js [hunk 2]
```diff
@@ -12,198 +12,213 @@ var Reply = postModels.Reply;
...
-		})
-	})
-
-	app.get('/api/notifications', function (req, res) { });
-
-	app.get('/*', function (req, res) {
-		res.sendfile('./public/404.html');
-	});
+    //api
+    app.get('/api/login/', function (req, res) {
```
Comment (iammosespaulr) [inline stackle_api/app/routes.js]: Don't mind the Quotes .. though codacy points it out .. but anyway use Double Quotes to pass the codacy test

Comment (bhavyaagg) [reply stackle_api/app/routes.js]: I have created PR #89 to solve the codacy issues.

**Comment by bhavyaagg** [comment]
I have created PR #89 to solve the codacy issues.

**Comment by bhavyaagg** [comment]
I have resolved the merge conflicts.

**Comment by bhavyaagg** [comment]
@psnmissaka @agentmilindu @padamchopra Can this be merged?

**Comment by psnmissaka** [comment]
resolve conflicts and update the PR

**Comment by bhavyaagg** [comment]
@psnmissaka Conflicts removed.

**Pull request merged**